flow/Platform.actor.cpp

```diff
@@ -3555,17 +3569,13 @@ void profileHandler(int sig) {
 
 	net2backtraces_offset += size + 2;
 #else
-	// No slow task profiling for other platforms!
+// No slow task profiling for other platforms!
 #endif
 }
 
 void setProfilingEnabled(int enabled) {
 #ifdef __linux__
-	if (profileThread && enabled && !profilingEnabled && profileRequested) {
-		profilingEnabled = true;
-		profileRequested = false;
-		pthread_kill(pthread_self(), SIGPROF);
```

Comment:
Removing deferred profiles is somewhat unrelated, but it kept thread local state and it has a problem in that we collect profiles when the thread sleeps for a long time.